app/services/books_service.rb

```diff
@@ -0,0 +1,12 @@
+class BooksService
+  include HTTParty
+  base_uri 'openlibrary.org'
```

Comment:
use credentials in this case, read this article https://blog.saeloun.com/2019/10/10/rails-6-adds-support-for-multi-environment-credentials.html